packages/@uppy/audio/src/RecordingScreen.js

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable jsx-a11y/media-has-caption */
+const { h, Component } = require('preact')
```

Comment:
Will we gradually switch to hooks? If so, might be good to do so with new plugins. Initial bundle size will increase a bit but that's inevitable if we would like to switch.

Reply:
Good point, converted to hooks.